flink-ai-flow/ai_flow_plugins/blob_manager_plugins/local_blob_manager.py

```diff
@@ -0,0 +1,91 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+import os
+import tempfile
+import zipfile
+from typing import Text, Dict, Any
+from pathlib import Path
+from ai_flow.plugin_interface.blob_manager_interface import BlobManager
+from ai_flow.util.file_util.zip_file_util import make_dir_zipfile
+
+
+class LocalBlobManager(BlobManager):
+    """
+    LocalBlobManager is an implementation of BlobManager based on the local file system.
+    LocalBlobManager contains 2 configuration items:
+    1. local_repository: It represents the root path of the downloaded project package.
+    2. remote_repository: It represents the root path of the upload project package.
+    """
+    def __init__(self, config: Dict[str, Any]):
```

Comment:
`LocalBlobManager` doesn't need the `local_repository` and `remote_repository` configuration items.